icu4c/source/common/unicode/localebuilder.h

```diff
@@ -276,6 +276,17 @@ class U_COMMON_API LocaleBuilder : public UObject {
      */
     Locale build(UErrorCode& status);
 
+    /**
+     * Sets the UErrorCode if an error occurred while recording sets.
+     * Preserves older error codes in the outErrorCode.
+     * @param outErrorCode Set to an error code if it does not contain one already
+     *                  and an error occurred while recording sets.
```

Comment:
Suggested documentation of `@param outErrorCode`:

```
Set to an error code that occurred while setting subtags.
Unchanged if there is no such error or if outErrorCode
already contained an error.
```

Reply:
DONE PTAL